api/config/config_perf.json

```diff
@@ -85,7 +88,7 @@
     "enableDataUseAgreement": true,
     "enableBetaAccess": true,
     "unsafeAllowSelfBypass": true,
-    "requireInvitationKey": true
+    "requireInvitationKey": false
```

Comment:
FYI @aweng98 in case this flag flip might affect our perf tests at all. I think it's unlikely to be impacted, just wanted to check.

(My thought process is that perf tests may be impacted only if both of the following are true: (1) nightly perf tests are creating new users, and (2) our API returns an error response if an invitation key is provided bu requireInvitationKey is false. My quick read of [this line](https://github.com/all-of-us/workbench/blob/master/api/src/main/java/org/pmiops/workbench/api/ProfileController.java#L343) is that we just ignore the invitation key when it's not required.

Reply:
I spoke to Alex offline - we're ok